python/giotto_json_reader.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
+
+import json
+import argparse
+
+
+def cells_json(data):
+    '''
+    >>> data = {
+    ...     "cell_1": {
+    ...         "locations": [1632.02, -1305.7],
+    ...         "factors": {
+    ...             "pleiden_clus": [3],
+    ...             "kmeans": [8]
+    ...         },
+    ...         "mappings": {
+    ...             "tsne": [-11.0776, 6.0311],
+    ...             "umap": [-6.858, 15.7691]
+    ...         }
+    ...     }
+    ... }
+    >>> cells = cells_json(data)
+    >>> cells.keys()
+    dict_keys(['cell_1'])
+    >>> cells['cell_1'].keys()
+    dict_keys(['mappings', 'genes', 'xy', 'factors', 'poly'])
+
+    '''
+    cell_dict = {}
+    for cell in data.keys():
```

Comment:
Preference for `cell_id`: With the loose typing, variable names probably should be more explicit in python than they are in a language like java.